taggit_bulk/actions.py

```diff
@@ -5,7 +5,7 @@
 from django.conf import settings
 from django.http import HttpResponseRedirect
 from django.urls import reverse
-from django.utils.translation import ugettext_lazy as _
+from django.utils.translation import gettext_lazy as _
```

Comment:
Please make a check against the python version here to be compatible with python2.* where the Unicode strings differ from strings, like:
```
import sys
if tuple(sys.version_info) < (3, 0, 0):
    from django.utils.translation import ugettext_lazy as _
else:
    from django.utils.translation import gettext_lazy as _
```

 The gettext_lazy may probably make erroneous output for python 2.* when using extended character codes.

Reply:
The last version of Django supports python >= 3.8. Shouldn't this extension follow the same pattern? In any case, I finished the requested changes.

Reply:
As you can see in the build script, this package supports Django versions from 1.10 which is compatible only with python 2.7. Django versions prior to 3 are also compatible with python 2.7, so we probably need to be compatible a while.

Thank you for a good job!